x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/nio/SSLDriverTests.java

```diff
@@ -159,20 +164,32 @@ public void testHandshakeFailureBecauseProtocolMismatch() throws Exception {
         SSLEngine clientEngine = sslContext.createSSLEngine();
         SSLEngine serverEngine = sslContext.createSSLEngine();
 
-        String[] serverProtocols = {"TLSv1.2"};
+        final String[] serverProtocols;
+        final String[] clientProtocols;
+        final String expectedMessage;
+        if (inFipsJvm()) {
+            // fips JSSE does not support TLSv1.3 yet
```

Comment:
mental note to revisit this when https://github.com/elastic/elasticsearch/issues/67233 is fixed, as 1.0.11 includes support for TLS1.3